src/SendGrid/SendGridClient.cs

```diff
@@ -50,8 +50,8 @@ public SendGridClient(HttpClient httpClient, string apiKey, string host = null,
         /// <param name="version">API version, override AddVersion to customize.</param>
         /// <param name="urlPath">Path to endpoint (e.g. /path/to/endpoint).</param>
         /// <returns>Interface to the Twilio SendGrid REST API.</returns>
-        public SendGridClient(string apiKey, string host = null, Dictionary<string, string> requestHeaders = null, string version = null, string urlPath = null)
-            : base(buildOptions(apiKey, host, requestHeaders, version, urlPath))
+        public SendGridClient(string apiKey, string host = null, Dictionary<string, string> requestHeaders = null, string version = null, string urlPath = null, bool httpErrorAsException = false)
```

Comment:
Think this constructor should be rolled back since it changes the signature and would require customers to re-compile. Maybe add a new constructor where `httpErrorAsException` is not optional?

Reply:
That's a good point. I thought the optional would take care of it, but I see that it does not.